gems/aws-sdk-sso/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Unreleased Changes
 ------------------
 
+1.7.0 (2020-09-01)
+------------------
+
+* Feature - SSO has been moved into `aws-sdk-core` to support usage of SSOCredentials.
```

Comment:
I'd mention here what version of core now has this change. Be careful though I think our changelog verifier test will look for "x.y.z" regex and fail (it's expecting to find 1.7.0 (2020-09-01) for instance but would find 3.105.0). I can't remember if I fixed that test or not, but worst case you could say 3.105.